README.md

```diff
@@ -1,17 +1,30 @@
+![logo](./static/icon.png)
+
 # Sunroof
 
-Sunroof is a [Vesktop](https://github.com/Vencord/Vesktop) fork
+[![Sunroof](https://img.shields.io/badge/Suncord-orange?style=flat)](https://github.com/verticalsync/Suncord)
+[![Tests](https://github.com/verticalsync/Sunroof/actions/workflows/test.yml/badge.svg?branch=main)](https://github.com/verticalsync/Sunroof/actions/workflows/test.yml)
+[![Discord](https://img.shields.io/discord/1207691698386501634.svg?color=768AD4&label=Discord&logo=discord&logoColor=white)](https://discord.gg/VasF3Ma4Ab)
 
-## Installing
+Sunroof is a fork of [Vesktop](https://github.com/Vencord/Vesktop)
 
-Find the required installer you need [here](https://github.com/verticalsync/Sunroof/releases/tag/latest).
+**Main features**:
+- Suncord preinstalled
+- Much more lightweight and faster than the official Discord app
+- Linux Screenshare with sound & wayland
+- Custom Splash animations from [this PR](https://github.com/Vencord/Vesktop/pull/355)
```

Comment:
Move this into a different header something like the suncord repo has about changes alongside vencord